PortalTests/TextViewRenderSpec.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TextViewRenderSpec.swift
+//  PortalTests
+//
+//  Created by Argentino Ducret on 8/28/17.
+//  Copyright © 2017 Guido Marucci Blas. All rights reserved.
+//
+
+import Quick
+import Nimble
+@testable import Portal
+
+class TextViewRenderSpec: QuickSpec {
+    override func spec() {
+    
+        var layoutEngine: LayoutEngine!
+        
+        beforeEach {
+            layoutEngine = YogaLayoutEngine()
+        }
+        
+        describe(".apply(changeSet: TextViewChangeSet) -> Result") {
+        
+            var changeSet: TextViewChangeSet!
+            
+            beforeEach {
+                let textViewStyle = textViewStyleSheet { base, textView in
+                    textView.textColor = .red
+                    textView.textFont = Font(name: "Helvetica")
+                    textView.textSize = 12
+                    textView.textAligment = .center
+                }
+                
+                changeSet = TextViewChangeSet.fullChangeSet(
+                    textType: .regular("This is a textView!"),
+                    style: textViewStyle,
+                    layout: layout())
+            }
+            
+            context("when the change set contains textType changes") {
+            
+                it("applies 'text' property changes") {
+                    let textView = UITextView()
+                    let _: Render<String> = textView.apply(changeSet: changeSet, layoutEngine: layoutEngine)
+                    expect(textView.text).to(equal("This is a textView!"))
+                }
+                
+                it("applies 'attributedText' property changes") {
+                    let textViewStyle = textViewStyleSheet { base, textView in
+                        textView.textColor = .red
+                        textView.textFont = Font(name: "Helvetica")
+                        textView.textSize = 12
+                        textView.textAligment = .center
+                    }
+                    
+                    changeSet = TextViewChangeSet.fullChangeSet(
+                        textType: .attributed(NSAttributedString(string: "This is a textView!")),
+                        style: textViewStyle,
+                        layout: layout()
+                    )
+                    
+                    let textView = UITextView()
+                    let _: Render<String> = textView.apply(changeSet: changeSet, layoutEngine: layoutEngine)
+                    
+                    expect(textView.attributedText.string).to(equal("This is a textView!"))
+                }
+                
+                
+                // TODO: Check NSAttributedString's attributes
```

Comment:
What about this?

Reply:
We should research a little about why the NSAttributedString has in some contexts an attribute called `NSOriginalFont`. I prefer keep advancing with other components and then check this.